src/nfa/operations.cc

```diff
@@ -270,6 +285,7 @@ Nfa Mata::Nfa::remove_epsilon(const Nfa& aut, Symbol epsilon) {
     return result;
 }
 
+//TODO: experiment with reverts and chose which should survive
 Nfa Mata::Nfa::fragile_revert(const Nfa& aut) {
```

Comment:
Side note: it is probably better to name it `revert_fragile`, then if you are using autocomplete, and write `revert` you will see better what options you have.